### DISCUSSION TITLE
where do you put web_fqdn in server chart and operator chart's values.yaml

### DISCUSSION THREAD

**Discussion by rwong2888** [kiali/kiali#4038] [Multicluster] [answered]
https://github.com/kiali/kiali/issues/3947#issuecomment-841166343

https://github.com/kiali/kiali-operator/blob/54e951ebea725a8ac7b9ae683c7ba1160d54990e/deploy/kiali/kiali_cr.yaml#L821-L828

**Comment by jmazzitelli** [comment, marked as answer]
I just created a PR to the kiali.io docs for this. See: https://github.com/kiali/kiali.io/pull/377

Here's a link to the new docs from that PR:

https://github.com/kiali/kiali.io/pull/377/files#diff-6ec9d44d36c1239297b78bd43b07e02eacc0c62db3fb27926ca3f5898c9c705aR169

**Reply by jmazzitelli** [reply]
Here's the netlify link - you can see the second lightbulb blurb with the docs from that PR: https://deploy-preview-377--kiali.netlify.app/documentation/staging/installation-guide/#_helm_chart

**Reply by rwong2888** [reply]
Thanks a bunch!
